src/Aquifer.Data/Entities/ResourceContentVersionEntity.cs

```diff
@@ -32,14 +32,11 @@ public class ResourceContentVersionEntity : IHasUpdatedTimestamp
 
     public ICollection<ResourceContentVersionMachineTranslationEntity> MachineTranslations { get; set; } = [];
 
-    public IEnumerable<ResourceContentVersionStatusHistoryEntity> ResourceContentVersionStatusHistories { get; set; } =
-        new List<ResourceContentVersionStatusHistoryEntity>();
+    public IEnumerable<ResourceContentVersionStatusHistoryEntity> ResourceContentVersionStatusHistories { get; set; } = [];
```

Comment:
This broke resource assignment.  See https://github.com/BiblioNexusStudio/aquifer-server/pull/503 for the fix.